static/demo.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```

Comment:
Yes, I can change it to split by character, but I think for English it might be better just to split by words.

Also, the metatags are from the default template created by VSCode (text editor).

I used Materialize just for the demo since I like it a bit better but you can definitely change that.

Reply:
Yeah that might be a good idea. People might write mixed things though, so that might be more tricky.

Okay cool, Materialize sounds good then!